README.md

```diff
@@ -402,3 +402,43 @@ $layout_registry = $view->getLayoutRegistry();
 $layout_registry->setTemplateFileExtension('.phtml');
 ?>
 ```
+
+### Advanced Configuration
+
+Alternatively you can pass [$helpers](https://github.com/harikt/Aura.View#custom-helper-managers),
+mapping information or paths for views and layouts as below.
+
+```php
+<?php
+$view_factory = new \Aura\View\ViewFactory;
+$view = $view_factory->newInstance(
+    $helpers = null,
```

Comment:
You are right, it can be just null. But I wanted to show it is helpers and people can pass helpers other than null.